app/controllers/kuroko2/api/job_definitions_controller.rb

```diff
@@ -0,0 +1,60 @@
+class Kuroko2::Api::JobDefinitionsController < Kuroko2::Api::ApplicationController
+  include Garage::RestfulActions
+
+  private
+
+  def require_resources
+  end
+
+  def create_resource
+    definition = Kuroko2::JobDefinition.new(definition_params(params))
+    user_ids = admin_id_params(params)
+    definition.admins = Kuroko2::User.active.with(user_ids)
+
+    if definition.save
+      definition.admins.each do |user|
+        user.stars.create(job_definition: definition) if user.google_account?
+      end
+      
+      @resource = Kuroko2::Api::JobDefinitionResource.new(definition)
+    else
+      raise WeakParameters::ValidationError.new("#{definition.name}: #{definition.errors.full_messages.join()}")
```

Comment:
It has nothing to do with `weak_parameters`.
Please use `Kuroko2::Api::ApplicationController#respond_with_error`.

Reply:
I have fixed it in f81b2ce.